frontend/components/RunCommand.jsx

```diff
@@ -0,0 +1,71 @@
+import { Box } from '@material-ui/core';
```

Comment:
Sorry if I was unclear, I didn't mean, merge the load chart and the command output into one widget, I simply meant refactor the code is such a way that you can use the command output formatting from the normal commands for displaying the output of the `uptime` command used for the load graph.